src/Models/Link.php

```diff
@@ -290,4 +292,23 @@ private function getLinkTypes(): array
 
         return $types;
     }
+
+    public function getDisplayTitle(): string
+    {
+        // If we have a title, we can just bail out without any changes
+        if ($this->Title && $this->getURL()) {
+            return $this->Title;
+        }
+        
+        $defaultLinkTitle = $this->getDefaultTitle();
+
+        $this->extend('updateDefaultLinkTitle', $defaultLinkTitle);
+
+        return $defaultLinkTitle;
+    }
+
+    public function getDefaultTitle(): string
+    {
+        return $this->getDescription() ?: $this->getURL();
```

Comment:
```suggestion
        $default = $this->getDescription() ?: $this->getURL();
        if (!$default) {
            $default = _t(static::class . '.MISSING_DEFAULT_TITLE', 'No link provided');
        }
        return $default;
```

This is important, because if there's no description or URL we still need to display _something_ as the default title to avoid an empty `<a>` tag. People can still change what that is through localisation and through the extension hook, but we _do_ need to provide some default title in this scenario.

See https://github.com/silverstripe/silverstripe-linkfield/pull/126#discussion_r1413227926